src/components/CommitHistory.tsx

```diff
@@ -192,8 +247,8 @@ export function CommitHistory() {
   )
 }
 
-function sortCommits(items: GitLogEntry[], method: CommitSortingMethodsType): { [key: string]: GitLogEntry[] } {
-  const cleanGroupItems: { [key: string]: GitLogEntry[] } = {}
+function sortCommits(items: FullCommitDTO[], method: SortCommitsMethods): { [key: string]: FullCommitDTO[] } {
+  const cleanGroupItems: { [key: string]: FullCommitDTO[] } = {}
   switch (method) {
     // case AUTHOR
     case Object.keys(SortingMethods)[1]:
```

Comment:
Is there a nicer method to do this? Maybe with a static string that is typed? Since method is already typed, a static string would be checked by typescript.

Reply:
idk, i think this is dawid code